data_structures/rb_tree2.cpp

```diff
@@ -0,0 +1,1728 @@
+/**
+ * @file
+ * @author [Brennan Allen](https://github.com/brennandoubt)
+ * (brennandoubt@gmail.com)
+ * @brief A Red-Black Tree (RBT) program with test cases.
+ * @version 0.1
+ * @date 2022-12-29
+ * 
+ * Copyright (no copyright)
+ * 
+ * @details A program simulating the Red Black Tree algorithm
+ * using helper functions with descriptive function names and annotations.
+ */
```

Comment:
```suggestion
/**
 * @file
 * @brief A Red-Black Tree (RBT) program with test cases.
 * @details A program simulating the Red Black Tree algorithm
 * using helper functions with descriptive function names and annotations.
 *
 * @author [Brennan Allen](https://github.com/brennandoubt)
 */
```
I think we should fit this to look like all the other files also i dont think doxygen accepts 2 links as a singular hyperlink